files/image_config/hostcfgd/hostcfgd

```diff
@@ -42,12 +42,13 @@ def obfuscate(data):
 
 
 def update_feature_state(feature_name, state, has_timer=False):
-    feature_suffix = "timer" if has_timer else "service"
+    feature_suffixes = ["service"] + (["timer"] if has_timer else [])
     if state == "enabled":
         start_cmds = []
-        start_cmds.append("sudo systemctl unmask {}.{}".format(feature_name, feature_suffix))
-        start_cmds.append("sudo systemctl enable {}.{}".format(feature_name, feature_suffix))
-        start_cmds.append("sudo systemctl start {}.{}".format(feature_name, feature_suffix))
+        for suffix in feature_suffixes:
+            start_cmds.append("sudo systemctl unmask {}.{}".format(feature_name, suffix))
+            start_cmds.append("sudo systemctl enable {}.{}".format(feature_name, suffix))
+        start_cmds.append("sudo systemctl start {}.{}".format(feature_name, feature_suffixes[-1]))
```

Comment:
Maybe add a comment here to describe that if a timer is present we want to start the time, not the service.